app/Resources/views/newsletter/invitation.html.twig

```diff
@@ -29,7 +29,7 @@
                 </p>
 
                 <div class="newsletter-page__form l__wrapper-500">
-                    {{ form_start(invitation_form) }}
+                    {{ form_start(invitation_form, {attr: {novalidate: 'novalidate'}}) }}
```

Comment:
I would much prefer to change the FormType to remove the "required" attribute. Is there a reason not to do so?